USERS.md

```diff
@@ -9,3 +9,4 @@ Please open a PR with your organization or project name in the following section
 
 1. [Dyno Therapeutics](https://www.dynotx.com)
 2. [Pipekit](https://www.pipekit.io)
+3. [ReverieLabs](https://www.reverielabs.com/)
```

Comment:
please add a space between Reverie and Labs

Reply:
Thank you and Reverie Labs, @dmerrick! ❤️